src/terminal7.js

```diff
@@ -858,7 +858,7 @@ export class Terminal7 {
             return
         if (gatePad) {
             const gate = gatePad.gate
-            const isExpand = e.classList.contains("expand-gate")
+            const isExpand = e.classList.contains("gate-edit")
```

Comment:
Thought we decided against renaming it? we're creating it with "expand-gate" like always